### PR TITLE
Add CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by SilleBille**
You can view a compiled version here: https://github.com/SilleBille/pki/blob/update-contribution/CONTRIBUTING.md

`Signed-off-by: Dinesh Prasanth M K <dmoluguw@redhat.com>`

### CODE REVIEW

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:93]: I think we discussed about renaming this page to "Contact Us".

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:97]: If possible add a reference to this page in this section:
https://github.com/dogtagpki/pki/blob/master/LICENSE
and avoid/minimize mentioning specific license names in this section so in case the license is updated this page won't become obsolete.

**Review by edewata — COMMENTED** [review]
Please see my comments above. Generally it looks good, but I'd wait until others have a chance to have a look a this doc as well. Also, I'd suggest renaming the commit/PR title to "Add CONTRIBUTING doc".

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md:93]: Hmm. I saw your suggestion for README. May be I missed. I will fix it now

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md]: Done

**Review by cipherboy — APPROVED** [review]
Good enough for now :)

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:97]: This is fine, but I'm thinking maybe it should say something like:

> All contributions must be submitted under the license specified in the [LICENSE](LICENSE) document.

I'll let you decide.

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:60]: Missing the ending period.

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:50]: Missing the ending period.

File: CONTRIBUTING.md
Comment (edewata) [inline CONTRIBUTING.md:16]: Missing the ending period.

**Review by edewata — APPROVED** [review]
The first line in the commit description still says "Add CONTRIBUTING markdown to the repo".
Other than that I just have a few minor comments below. Feel free to merge if others do not have any other comments.

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md
Comment (SilleBille) [reply CONTRIBUTING.md]: Done

**Comment by SilleBille** [comment]
Thanks for the review @edewata and @cipherboy :+1: . Merging based on ACKs and internal discussions that happened via GDoc. We can always update this doc in future, if required.

Merging it without waiting for CI since this is a doc change :wink:

**Pull request merged**
